app/controllers/api/v1/skills_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::SkillsController < Api::V1::BaseController
+  respond_to :json
+
+  def index
+    featured = params[:featured]
+    respond_with featured ? Skill.where(featured: featured) : Skill.all
+  end
+
```

Comment:
Just a question here: What's the reason for using classes instead of module in this namespacing/inheritance? I think that's the way we've always seen it, but I'm curious--modules are designed just for sharing functionality when you don't need to instantiate a new instance. I'm not suggesting a change per se, just wondering if there is any reason to use classes over modules in this situation.